python/activator/middleware_interface.py

```diff
@@ -624,21 +623,16 @@ def _export_refcats(self, region):
         refcats : iterable [`DatasetRef`]
             The refcats to be exported, after any filtering.
         """
-        center = lsst.geom.SpherePoint(region.getCentroid())
-        radius = max([center.separation(lsst.geom.SpherePoint(vertex)) for vertex in region.getVertices()])
-        indexer = HtmIndexer(depth=7)
-        shard_ids, _ = indexer.getShardIds(center, radius)
-        htm_where = f"htm7 in ({','.join(str(x) for x in shard_ids)})"
         # Get shards from all refcats that overlap this region.
         possible_refcats = _get_refcat_types(self.central_butler)
-        _log.debug("Searching for refcats of types %s in %s...",
-                   {t.name for t in possible_refcats}, shard_ids)
+        _log.debug("Searching for refcats of types %s.", {t.name for t in possible_refcats})
         refcats = set(_filter_datasets(
                       self.central_butler, self.butler,
                       possible_refcats,
                       collections=self.instrument.makeRefCatCollectionName(),
-                      where=htm_where,
-                      findFirst=True,
+                      where="htm7.region OVERLAPS search_region",
```

Comment:
Should this be `htm7`, or `skypix`? I'm not clear on the arcane rules for when the latter is allowed.

Reply:
- Naively trying `skypix` gave me `Unrecognized identifier 'skypix.region'.` 
- I _think_ it has to be a spatial dimension with a region field
- `skypix` is not a dimension.  `htm7`/`patch`/`tract` are, and they all work to some degree.  Picking `htm7` based on this [Slack thread discussion ](https://lsstc.slack.com/archives/C01FBUGM2CV/p1729109758075629)